source/exe/process_wide.cc

```diff
@@ -31,9 +31,13 @@ ProcessWide::ProcessWide(bool validate_proto_descriptors) {
     // TODO(mattklein123): Audit the following as not all of these have to be re-initialized in the
     // edge case where something does init/destroy/init/destroy.
     Event::Libevent::Global::initialize();
+#ifdef ENVOY_ENABLE_FULL_PROTOS
     if (validate_proto_descriptors) {
       Envoy::Server::validateProtoDescriptors();
     }
+#else
+    (void)validate_proto_descriptors;
```

Comment:
what is the purpose of this line?

Reply:
Oh sorry. It makes the compiler not complain about the parameter being unused. Switched to the Envoy idiom `UNREFERENCED_PARAMETER()`. Thanks!